treeherder/etl/tasks/pulse_tasks.py

```diff
@@ -16,7 +17,14 @@ def store_pulse_jobs(pulse_job, exchange, routing_key):
     newrelic.agent.add_custom_parameter("exchange", exchange)
     newrelic.agent.add_custom_parameter("routing_key", routing_key)
 
-    JobLoader().process_job(pulse_job)
+    # handleMessage expects messages in this format
+    tc_th_message = handleMessage({
+        "exchange": exchange,
+        "routes": routing_key,
```

Comment:
this is a bit confusing, too -- the key is plural but its value is singular

Reply:
I removed the need to pass `routes` in general.